src/core/task10.js

```diff
@@ -1,13 +1,20 @@
-export var obj = {
-      first: prompt("Введите первое число"),
+export default function Number() {
 
-      second: prompt("Введите второе число"),
+    this.first = prompt("Введите первое число");
 
-      plus: function() {
-        return (+this.first) + (+this.second);
-      },
+    this.second = prompt("Введите второе число");
```

Comment:
We need a function that will launch a process of storing values (first & second)
For example: 
1. Create an object
2. Call the function that will ask about all necessary values. 
2. Call the function that will display these values' summary in an alert. 
4. Call the function that will display these values' multiply in an alert.